src/styles/blocks/recommended.scss

```diff
@@ -0,0 +1,126 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.recommended {
+  @include set-horizontal-padding;
+
+  &__title {
+    width: 100%;
+    margin-top: $site-parts-top-margin;
+    text-align: center;
+
+    @include style-title;
+  }
+
+  &__soundbars {
+    @include for-desktop {
+      grid-template-rows: repeat(2, max-content);
+
+      @include set-desktop-grid;
+    }
+  }
+
+  &__picture {
+    display: block;
+    margin: 48px auto 0;
+
+    @include increase-hover-scale;
+
+    @include for-tablet {
+      margin-top: 60px;
+
+      &:nth-of-type(1) {
+        content: url("../../src/images/photos/soundbars/smart/smart-744.png");
```

Comment:
I would rather put those images into html as these also represent bose's products, also it is better to differentiate the declarative layer of what is on your page from the styling

Reply:
I reorganized images to only one with manipulate sizes in css.